weblate/addons/tasks.py

```diff
@@ -176,3 +176,38 @@ def setup_periodic_tasks(sender, **kwargs) -> None:
         cleanup_addon_activity_log.s(),
         name="cleanup-addon-activity-log",
     )
+
+
+@app.task(trail=True)
+def addon_change(sender, change_ids: list[int], **kwargs) -> None:
```

Comment:
This is our interface, so it should be strictly defined:

```suggestion
def addon_change(change_ids: list[int]) -> None:
```